playground/buffer-mesh-particle-shader-effect.ts

```diff
@@ -0,0 +1,237 @@
+import {
```

Comment:
Add comments, **title** and **category**.

ref: https://github.com/ant-galaxy/oasis-engine.github.io/blob/main/playground/buffer-mesh-independent.ts

Reply:
handled